src/AutoVizuA11y.tsx

```diff
@@ -265,6 +291,18 @@ const AutoVizuA11y = ({
 		asyncEffect();
 	}, [chartRef]);
 
+	if (!isValidElement(shortcutGuide)) {
```

Comment:
Are my eyes deceiving me or is this duplicated?
Also, we can clean this up a bit by moving the conditional logic to a single constant and then using the ternary operator to do this in a single line.

Like so:

```js
const IS_SHORTCUT_GUIDE_ELEMENT = !!(isValidElement(shortcutGuide));

shortcutGuide = IS_SHORTCUT_GUIDE_ELEMENT
        ? cloneElement(shortcutGuide, { closeShortcutGuide })
        : <ShortcutGuide closeShortcutGuide={closeShortcutGuide} />
```